skimage/transform/radon_transform.py

```diff
@@ -149,36 +153,40 @@ def _get_fourier_filter(size, filter_name):
     ----------
     .. [1] AC Kak, M Slaney, "Principles of Computerized Tomographic
            Imaging", IEEE Press 1988.
-
     """
-    n = np.concatenate((np.arange(1, size / 2 + 1, 2, dtype=np.int),
-                        np.arange(size / 2 - 1, 0, -2, dtype=np.int)))
+    # Construct the Fourier filter
+
+    # The ramp filter is implemented based on eq. 61 on  p.72 of Kak and Slaney
+    # https://engineering.purdue.edu/~malcolm/pct/CTI_Ch03.pdf
+    n1 = np.arange(0, size / 2 + 1, dtype=np.int)
+    n2 = np.arange(size / 2 - 1, 0, -1, dtype=np.int)
+    n = np.concatenate((n1, n2))
     f = np.zeros(size)
     f[0] = 0.25
-    f[1::2] = -1 / (np.pi * n) ** 2
+    f[1::2] = -1 / (np.pi * n[1::2])**2
```

Comment:
was it a bug which you corrected? I don't understand why f is zero on even values of indices, this does not correspond to Kak and Slaney I think.

Reply:
ok I took a look at one of the previous PRs and apparently it's not the classical ramp filter which is used here. I'll try to understand better which filter is used here. Did you just make this change for the sake of a clearer syntax or is there a difference?

Reply:
I think at the time, the change was introduced for cleaner syntax.

Reply:
The "fixes" for the filter performance were merged in a pervious PR.

Reply:
Your current refactoring of `n` calculation is different from previous one:
```python
>>> size = 10
>>> n1 = np.arange(0, size / 2 + 1, dtype=np.int)
>>> n2 = np.arange(size / 2 - 1, 0, -1, dtype=np.int)
>>> n = np.concatenate((n1, n2))
>>> n[1::2]  # This PR refactoring
array([1, 3, 5, 3, 1])
>>> np.concatenate((np.arange(1, size / 2 + 1, 2, dtype=np.int),
                    np.arange(size / 2 - 1, 0, -2, dtype=np.int)))  # Previous version
array([1, 3, 5, 4, 2])
```
I don't know which version is correct, but there is a difference...